packages/renderer/src/lib/kubernetes-port-forward/PortForwardIcon.svelte

```diff
@@ -0,0 +1,20 @@
+<script lang="ts">
+import { faEthernet } from '@fortawesome/free-solid-svg-icons';
```

Comment:
unsure regarding using faEthernet, wdyt @deboer-tim

Reply:
I don't have any icon provided nor mock-ups

Reply:
Let's use faEthernet for the time being.